spec/controllers/api/v1/marketing/lists_controller_spec.rb

```diff
@@ -0,0 +1,250 @@
+# bundle exec rspec --format documentation spec/controllers/api/v1/marketing/lists_controller_spec.rb 
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::Marketing::ListsController do
+
+    # 
+    # initialize()
+    # 
+    def initialize (x)
+        @base_earl = 'api/marketing/lists?'
+        @bugger = false
+    end
+
+    # 
+    # show_test()
+    # 
+    def show_test (t, p)
+        puts ''
+        puts '  trying ' + @base_earl + p.to_query
+        puts '    [' + t + ']'
+    end
+
+    # 
+    # show_subtest()
+    # 
+    def show_subtest (message, result = true)
+        printf "    %-80s %s\n", message, (result ? '[OK]' : '*** FAILED ***')
+    end
+
+    # 
+    # bugger_add()
+    # 
+    def bugger_add (message)
+        pp message if @bugger
+    end
+
+    # 
+    # describe
+    # 
+    describe 'GET get_list' do
+
+        # get with no params (code 1, method not allowed)
+        t1 = 'get with no params (code 1, method not allowed)'
+        it t1 do
+            params = {}
```

Comment:
Yeah, but I wanted to reuse the string, And I'm somehow not capable of repeating a string instead of putting it in a variable. I know it's not typical, but if I'm testing something I want to see what is being tested and the result. For a moment I thought about hijacking stdout to get rid of the dots, but then decided maybe that was a bit too far.